refinery/tool_manager/models.py

```diff
@@ -469,9 +469,13 @@ def _get_visualization_parameters(self):
                 {
                     "uuid": parameter.uuid,
                     "description": parameter.description,
-                    "default_value": parameter.default_value,
+                    "default_value": parameter.cast_param_value_to_proper_type(
+                        parameter.default_value
```

Comment:
Again, I'm a little confused: I had thought this was moving in a direction to support native types?

Reply:
(Scott explained in person: This is coming from the database, where everything is string.)